SingleCellPatch/extract_patches.py

```diff
@@ -38,6 +36,12 @@ def select_window(mat, window, padding=0.):
         np.array: submatrix-of-interest
     
     """
+    if padding is None and ((window[0][0] < 0) or
```

Comment:
I guess the purpose is here is to neglect all patches that exceed the boundary. I suggest setting up a flag and let pipeline  caller functions to decide. (I will make this change on my PR)

Reply:
Please check the flag `skip_boundary` in PR #17

Reply:
Sounds good. We can use your implementation.